omp/factorization/factorization_kernels.cpp

```diff
@@ -231,16 +231,13 @@ void initialize_row_ptrs_l_u(
         bool has_diagonal{};
```

Comment:
```suggestion
```

Reply:
The one that got away :laughing: thanks!